client_it_test.go

```diff
@@ -179,6 +180,67 @@ func TestSetRowAttrs(t *testing.T) {
 	}
 }
 
+func TestOrmCount(t *testing.T) {
+	client := getClient()
+	countFrame, err := index.Frame("count-test", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = client.EnsureFrame(countFrame)
+	if err != nil {
+		t.Fatal(err)
+	}
+	qry := index.BatchQuery(
+		countFrame.SetBit(10, 20),
+		countFrame.SetBit(10, 21),
+		countFrame.SetBit(15, 25),
+	)
+	client.Query(qry, nil)
+	response, err := client.Query(index.Count(countFrame.Bitmap(10)), nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if response.Result().Count != 2 {
+		t.Fatalf("Count should be 2")
+	}
+}
+
+func TestTopNReturns(t *testing.T) {
+	client := getClient()
+	frame, err := index.Frame("topn_test", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = client.EnsureFrame(frame)
+	if err != nil {
+		t.Fatal(err)
+	}
+	qry := index.BatchQuery(
+		frame.SetBit(10, 5),
+		frame.SetBit(10, 10),
+		frame.SetBit(10, 15),
+		frame.SetBit(20, 5),
+		frame.SetBit(30, 5),
+	)
+	client.Query(qry, nil)
+	time.Sleep(10 * time.Second)
```

Comment:
I've added a comment for other clients but missed this one. Thanks for noticing. I think it would be great to be able to cause the Pilosa server to invalidate caches/flush the data. Apart from testing, that would be very desirable when/if we incorporate 3rd party backup solutions. I was thinking maybe we could send a special signal to the server which causes invalidation/flush (but that isn't easily usable by tests).